### ISSUE TITLE
[Bug] Sub-application layouts do not work?

### ISSUE THREAD

**Issue by Guard13007** [leafo/lapis#472]
I assumed I would be able to have a `views/layout.moon` for example, and then within a sub-application, declare that as the layout with `layout: require "views.layout"`, but that doesn't work.

What other kinds of things don't work with sub-applications the way they do with a master application? Or did I mess something up?

**Comment by leafo** [comment]
sub applications work by copying everything into the parent application, but optionally scoping each action's name/path with the parameters provided in the layout. Before filters are applied before copying as well, so they should work as expected

**Comment by Guard13007** [comment]
So, name/path and before filters are the only features that work with sub-applications?

**Comment by leafo** [comment]
yup https://github.com/leafo/lapis/blob/master/lapis/application.moon#L208
